task/tekton-catalog-publish/0.1/tests/run.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: catalog-publish-test-pipeline
+spec:
+  workspaces:
+  - name: shared-workspace
```

Comment:
I think twice-indented arrays is the norm in this repo? I.e.
```yaml
foo:
  - bar
```

Reply:
done